src/Microsoft.VisualBasic/ref/Microsoft.VisualBasic.csproj

```diff
@@ -5,6 +5,11 @@
     <AssemblyVersion>10.0.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
     <NuGetTargetMoniker>.NETStandard,Version=v1.1</NuGetTargetMoniker>
+    <!-- Xamarin frameworks claim to support PCL profiles which include this assembly, 
+         but do not include this assembly in the targeting pack.
+         This causes NuGet to give them the placeholder from the portable-* folder.
+         Instead give them this assembly.  -->
+    <PackageTargetFramework>netstandard1.1;MonoAndroid10;MonoTouch10;xamarinios10;xamarintvos10;xamarinwatchos10</PackageTargetFramework>
```

Comment:
We do have one, but its only in the packaging targets which are only imported in pkgproj's.  I'd prefer not to duplicate that.  Even further I'd prefer if these assemblies were just copied into Xamarin's frameworks so we don't have to special case like this.  Both this and System.Reflection.Context are part of PCL profiles which they support but the assemblies are missing from their targeting pack and framework.